lib/data/external/GCP/GcpApis/uploadPart.js

```diff
@@ -17,9 +17,16 @@ function uploadPart(params, callback) {
         logHelper(logger, 'error', 'error in uploadPart', error);
         return callback(error);
     }
+    const partNumber = getPartNumber(params.PartNumber);
+    if (!partNumber) {
+        const error = errors.InvalidArgument
+            .customizeDescription('PartNumber is not a number');
```

Comment:
Since `!partNumber` would mean 0 is invalid part number, it would be more accurate to customize description as `Part Number is invalid`.